tornjak-frontend/src/components/tornjak-api-helpers.js

```diff
@@ -151,7 +151,7 @@ class TornjakApi extends Component {
   // populateServerInfo returns the server trust domain and nodeAttestorPlugin
   populateServerInfo = (serverInfo, serverInfoUpdateFunc) => {
     //node attestor plugin
-    if (serverInfo === "" || serverInfo === undefined) {
+    if (serverInfo === "" || serverInfo === undefined || JSON.stringify(serverInfo) === '{}') {
```

Comment:
Comparison with empty string or undefined not needed. alternative
```
if (!serverInfo || JSON.stringify(serverInfo) === '{}') {
```
or 
```
if (!serverInfo || (serverInfo && !Object.keys(serverInfo).length)) {
```

Reply:
File not of concern on this PR. Will be addressed in the next consecutive PR.